pkg/controller/tenant/tenant_controller.go

```diff
@@ -355,3 +357,20 @@ func flattenedError(failures []error, tenant string) (error, bool) {
 	}
 	return nil, true
 }
+
+// not returning any error as the system should continue run without an explicit system tenant
+func (tc *TenantController) cerateSystemTenantIfNotExist() {
+	_, err := tc.lister.Get(metav1.TenantSystem)
+
+	if apierrors.IsNotFound(err) {
+		tc.kubeClient.CoreV1().Tenants().Create(&v1.Tenant{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: metav1.TenantSystem,
+			},
+			Spec: v1.TenantSpec{
+				StorageClusterId: "system",
```

Comment:
do we have some constant for the string "system" somewhere? it's used repeatedly in lots of code.

Reply:
I had a talk with Ying Huang on this a while. According to her, we can just hard code "system" for now. She will define a constant later. Issue (https://github.com/futurewei-cloud/arktos/issues/326) was created and assigned to her to track this.